lightnode.go

```diff
@@ -22,26 +27,35 @@ type Lightnode struct {
 }
 
 // NewLightnode constructs a new Lightnode.
-func NewLightnode(logger logrus.FieldLogger, cap, workers, timeout int, port string, addresses []string) *Lightnode {
+func NewLightnode(logger logrus.FieldLogger, cap, workers, timeout int, port string, bootstrapAddrs []peer.MultiAddr) *Lightnode {
```

Comment:
This function can just be called `New` because it lives in `package lightnode` and no other resource is created from this package. This means that package users call `lightnode.New()` which is more concise than `lightnode.NewLightnode()`.